docs/app/references/migration-guide.mdx

```diff
@@ -108,6 +108,46 @@ it('renders', () => {
 })
 ```
 
+### Angular versions under `17.2.0` are no longer supported in Component Testing.
+
+With [LTS ending](https://angular.dev/reference/releases#actively-supported-versions) for Angular 16, the minimum required Angular version for component testing is now `17.2.0` in order to support [signals](/app/component-testing/angular/examples#Signals) as a first class citizen.
+
+If for any reason an older version of Angular must be used, please download [`@cypress/angular`](https://www.npmjs.com/package/@cypress/angular) `2.x` from `npm`.
+
+```sh
+npm install --save-dev @cypress/angular@2
+```
+
+Then, import the `mount` from from this package in place of `cypress/angular`.
+
+```ts
+import { mount } from `cypress/angular`
+```
+
+to
+
+```ts
+import { mount } from `@cypress/angular`
+```
+
+This version of `@cypress/angular` is not actively supported and is only recommended as a workaround until you are able to upgrade.
+
+### `cypress/angular` now supports signals. The `cypress/angular-signals` test harness has been removed from the binary.
+
+Now that the minimum version of Angular supported for Component Testing is `17.2.0`, Cypress is able to merge the `cypress/angular-signals` test harness into the main `cypress/angular` test harness. Because of this, the `@cypress/angular-signals` harness is deprecated and no longer shipped with the binary. Support has been moved to `@cypress/angular`.
```

Comment:
```suggestion
Now that the minimum version of Angular supported for Component Testing is `17.2.0`, Cypress is able to merge the `cypress/angular-signals` test harness into the main `cypress/angular` test harness. Because of this, the `@cypress/angular-signals` harness is deprecated and no longer shipped with the binary. Support has been moved to `cypress/angular`.
```